scripts/generate-api-protos.sh

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env bash
```

Comment:
May be this aspect is already handled: The developers are supposed to make changes to the proto files and commit the generated files in the PR. What happens if they don't. Can we put some check such that there are no git diffs in the generated proto files?

Reply:
Good call, I added a script to check for this.